tests/adapters/test_adapter.py

```diff
@@ -37,3 +37,9 @@ def test_adapter_values():
     assert a.values("X,Y,Z") == ['X', 'Y', 'Z']
     assert a.values("X,Y,Z", cast=str) == ['X', 'Y', 'Z']
     assert a.values("X.Y.Z", separator='.') == ['X', 'Y', 'Z']
+
+
+def test_adapter_preprocess_reply():
```

Comment:
Thanks for the test. Can you please also add tests for the other possible constellations to confirm they work correctly? The following come to my mind, but maybe I am missing some
* adapter-level only (done)
* adapter-level, overridden by values kwarg
* values-kwarg only
* values-kwarg None ensures adapter-level only does not run.

Can you please also add one showing how an Instrument `control` uses this and confirm correct operation?

Reply:
> values-kwarg None ensures adapter-level only does not run.
This is against the behavior of the code where None is the default value of the `preprocess_reply` values-kwarg. To deactivate the adapter level one currently has to specify a real noop-function like `lambda v: v`. (Maybe one disadvantage when one explicitly includes the kwarg in the call signature.) Is this behavior sufficient, or you have ideas?

Reply:
Ah, didn't think of that, of course that doesn't work. Yes, a noop lambda seems like the best approach, we just need to document this so folks know how to achieve disabling the preprocessor where needed.